src/Command/CommandBase.php

```diff
@@ -1175,11 +1175,11 @@ protected function checkForNewVersion(InputInterface $input, OutputInterface $ou
       return;
     }
     // Bale in Cloud IDEs to avoid hitting Github API rate limits.
-    if (!AcquiaDrupalEnvironmentDetector::isAhIdeEnv()) {
```

Comment:
@anavarre 🤔 Seems important to fix.

Reply:
Without this IDEs were constantly asking to update and customers were doing it, which had us run into API limits without a few hours of an ACLI release.

Reply:
I'm not removing this line, I'm removing the `!`

Reply:
When I acli `master` in an IDE, the code does (incorrectly) progress to execute `$this->hasUpdate()`.